en/docs/install-and-setup/setup/distributed-deployment/configuring-wso2-identity-server-as-a-key-manager.md

```diff
@@ -309,6 +309,10 @@ Start WSO2 Identity Server for the changes to take effect. For more information,
 
         To overcome this issue, you need to create self-signed certificates for WSO2 API-M and WSO2 IS hostnames. Then [import the public certificates]({{base_path}}/install-and-setup/setup/security/configuring-keystores/keystore-basics/creating-new-keystores/#step-3-importing-certificates-to-the-truststore) of WSO2 API-M to the `trust-store.jks` of WSO2 IS and vice versa. This should resolve the SSL handshake failure.
 
+    !!! Note
+        To invoke RESTful APIs (product APIs), users must generate tokens through API Manager token endpoint.
```

Comment:
```suggestion
        To invoke RESTful APIs (product APIs), users must generate tokens through API Manager (Control Plane) token endpoint.
```

Reply:
Ack. Fixed in https://github.com/wso2/docs-apim/compare/a251be7d579a1dcdf3571054a396e8f82243eaf7..65eae6dea6ae65e3e1c0136dfbd4342dc099cf59